ansible/files/runtimes-nodeonly.json

```diff
@@ -1,4 +1,18 @@
 {
+    "description": [
+        "This file describes the different languages (aka. managed action runtimes) supported by the system",
+        "as well as blackbox images that support the runtime specification.",
+        "Only actions with runtime families / kinds defined here can be created / read / updated / deleted / invoked.",
+        "Define a list of runtime families (example: 'nodejs') with at least one kind per family (example: 'nodejs:10').",
+        "Each runtime family needs a default kind (default: true).",
+        "When removing or renaming runtime families or runtime kinds from this file, all already existing actions",
```

Comment:
Wordsmithing: 
        "When removing or renaming runtime families or runtime kinds from this file, all preexisting actions",
        "with the affected kinds can no longer be read / updated / deleted / invoked. In order to remove or rename

Reply:
Thanks a lot for providing more accurate wording / terms.